src/main/java/fr/xephi/authme/service/SpectateLoginService.java

```diff
@@ -0,0 +1,86 @@
+package fr.xephi.authme.service;
+
+import fr.xephi.authme.settings.properties.RestrictionSettings;
+import org.bukkit.GameMode;
+import org.bukkit.Location;
+import org.bukkit.entity.ArmorStand;
+import org.bukkit.entity.Player;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SpectateLoginService {
```

Comment:
Good name! Allows us to remember that this will set players to SPECTATOR game mode :)

Reply:
Added a description of the class behavior